### ISSUE TITLE
Where is the blobcopy.ps1?

### ISSUE THREAD

**Issue by recintoit** [Azure/migAz#102]
Is this supposed to be included in the downloaded solution?  I don't see it in the .zip.

Thank you,

**Comment by rsanderson2350** [comment]
Blobcopy.ps1 has become MigAz.ps1 as of v2.3.0.0.  You should see it in the output folder to which you generate the export resources.

**Comment by recintoit** [comment]
Thanks,
Could you answer a quick question?  I've done up to the part where MigAz
copied the blob and added the URI to the parameters JSON.  However I am
getting an error that the value is invalid.  I checked and it is the
correct value.

**Comment by rsanderson2350** [comment]
Could you provide an screenshot of what you are seeing in the PowerShell output as well as the CopyBlobDetails?  Please do filter out / remove any private details such as organization names, storage account name, any access keys, etc.  It would help to have an understanding what you are seeing.

**Comment by recintoit** [comment]
Here is the screenshot of the PS output.  And the BlobCopy output.  I removed keys, but it's ok because I am working in the MSDN before moving to an Enterprise Sub.
[NewResourceGroup_BlobCopy-Copy.txt](https://github.com/Azure/migAz/files/1641179/NewResourceGroup_BlobCopy-Copy.txt)

![troubleshoot_migaz](https://user-images.githubusercontent.com/35378940/35075317-3f458140-fba7-11e7-86ea-035efa716376.JPG)

**Comment by recintoit** [comment]
I downloaded v2.3.0.1 and tried again.  It seemed to work better.  I did change some options this time too. I'm not sure if that had anything to do with it  This actually did it all in one pass!  I will keep looking but it is promising.  Thank you
![successmigaz](https://user-images.githubusercontent.com/35378940/35132402-e9ff7d74-fc7f-11e7-94c4-7f69dd838574.JPG)

**Comment by rsanderson2350** [comment]
Glad to hear it is working for you.  I started replying earlier today but didn't submit the comment as I was starting to think there was a specific case I needed to dig deeper into.  You have arrived at the expected result, in which the BlobCopy parameter is replaced with the URL in the temporary MigAz Storage Account:

<img width="1098" alt="capture" src="https://user-images.githubusercontent.com/16927960/35138797-7181c0f4-fcad-11e7-9261-031e38d1360f.PNG">

I'll go ahead and close the issue again.  If you have any other problems, go ahead and open a new issue (and you reference back to this issue # if needed / if similar).
